great_expectations/checkpoint/actions.py

```diff
@@ -119,6 +122,9 @@ class Config:
     type: str
     notify_on: Literal["all", "failure", "success"] = "all"
 
+    def __lt__(self, other: ValidationAction) -> bool:
+        return self._sorting_priority < other._sorting_priority
```

Comment:
YAGNI? I preferred this to an explicit "hey data docs action is the very specific case we have to call out"

Reply:
Yeah, I'd just pulled out the special data docs actions now and put it at the top, leaving the rest unchanged. I don't know if we need/want general sorting.